src/mame/sharp/mz80_m.cpp

```diff
@@ -25,15 +25,12 @@ void mz80_state::machine_reset()
 	m_mz80k_keyboard_line = 0;
 }
 
-
 uint8_t mz80_state::mz80k_8255_portb_r()
```

Comment:
Please put the double blank line separating lifecycle from I/O back.

Reply:
I think you've done it?

Reply:
Yeah, I’m in a hurry.